turbopack/crates/turbopack-core/src/module_graph/mod.rs

```diff
@@ -1066,7 +1099,7 @@ impl Visit<SingleModuleGraphBuilderNode> for SingleModuleGraphBuilder<'_> {
             Ok(match (module, chunkable_ref_target) {
                 (Some(module), None) => {
                     let refs_cell = primary_chunkable_referenced_modules(*module);
-                    let refs = refs_cell.await?;
+                    let refs = refs_cell.await.context(module.ident().to_string().await?)?;
```

Comment:
Could we avoid calling .ident().to_string() when there is no error? (using match)